doc/en/components/charts/chart-types-bar.md

```diff
@@ -6,23 +6,118 @@ mentionedTypes: ["XamDataChart"]
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
 
 ## $PlatformShort$ Stacked Bar Chart
 
-<!-- TODO copy and combine content (code snippets, description) from these topics:
-	data-chart-type-stacked-bar-series.md
-    data-chart-type-stacked-100-bar-series.md
--->
+A Stacked Bar Chart, or Stacked Bar Graph, is a type of category chart that is used to compare the composition of different categories of data by displaying different sized fragments in the horizontal bars of the chart. The length of each bar, or stack of fragments, is proportionate to its overall value. 
+
+The $PlatformShort$ Stacked Bar Chart differs from the $PlatformShort$ Bar Chart in that the data points representing your data are stacked next to each other horizontally to visually group data your data. Each stack can contain both positive and negative values. All positive values are grouped on the positive side of the X-Axis, and all negative values are grouped on the negative side of the X-Axis.
+
+### $PlatformShort$ Stacked Bar Chart Example 
+
+In this example of an $PlatformShort$ Stacked Bar Chart, we have a Numeric X Axis (bottom labels of the chart) and a Category Y Axis (left labels of the chart).
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+## $PlatformShort$ Stacked 100 Bar Chart
+
+The $PlatformShort$ Stacked 100 Bar Chart is identical to the $PlatformShort$ stacked bar chart in all aspects except in their treatment of the values on X-Axis (bottom labels of the chart). Instead of presenting a direct representation of the data, the stacked 100 bar chart presents the data in terms of percent of the sum of all values in a data point.
+
+### $PlatformShort$ Stacked 100 Bar Chart Example 
+
+In this example of a Stacked 100 Bar Chart, the Energy Product values are shown as a 100% value of all of the data in the fragments of the horizontal bars.
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+## $PlatformShort$ Range Bar Chart
+
+The Ignite UI for $PlatformShort$ range column chart belongs to a group of range charts and is rendered using two lines with the area between the lines filled in. This type of series emphasizes the amount of change between low values and high values in the same data point over a period of time or compares multiple items. Range values are represented on the y-axis and categories are displayed on the X-Axis.
+
+### $PlatformShort$ Range Bar Chart Example 
+
+In this Range Bar Chart example, weather data variance between high and low temperatures by date is shown.
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
```

Comment:
The given example in the Word document is a Range **COLUMN** Chart. Is it right to use it in this topic? Maybe we need a different scenario and example?

Reply:
I think you are correct, maybe we need to move this section there and change Bar for Column

Reply:
@StefanIvanov @tosharky which samples and data do you want to show in this section? BTW, I would remove "Bar" from title of this section because we already have Bar Chart

Reply:
@HUSSAR-mtrela @jberes @pamelabrasil
This is a Range Bar Chart: https://www.anychart.com/products/anychart/gallery/Range_Charts/Range_Bar_Chart.php

We currently don't have that kind of chart in our documentation and samples. If we are going to provide it I will design it, no problem. But it wasn't on the list of charts to be re/designed, so I am not sure if it was just a mistake with the Column chart, that we have?

Reply:
yes, you are correct we do not have this kind of chart so this section should be removed because  we cannot create this it